ckanext/superset/tests/test_superset_tab.py

```diff
@@ -0,0 +1,235 @@
+class TestSupersetViews:
+    """Tests para las vistas de Superset"""
+
+    def test_superset_settings(self):
+        """Verifica si la configuración de Superset se está cargando correctamente"""
+        config_keys = [
+            "ckanext.superset.instance.url",
+            "ckanext.superset.instance.user",
+            "ckanext.superset.instance.pass",
+            "ckanext.superset.instance.provider",
+            "ckanext.superset.instance.refresh",
+        ]
+
+        for key in config_keys:
+            value = toolkit.config.get(key, "NO ENCONTRADO")
+            print(f"{key}: {value}")
+
+        assert toolkit.config.get("ckanext.superset.instance.url") is not None, "URL de Superset no encontrada"
+        assert toolkit.config.get("ckanext.superset.instance.user") is not None, "Usuario de Superset no encontrado"
+        assert toolkit.config.get("ckanext.superset.instance.pass") is not None, "Contraseña de Superset no encontrada"
+        assert toolkit.config.get("ckanext.superset.instance.provider") is not None, "Proveedor de Superset no encontrado"
+        text = "Configuración de actualización de Superset no encontrada"
+        assert toolkit.config.get("ckanext.superset.instance.refresh") is not None, text
+
+    def test_index_sysadmin_can_access(self, app_httpx_mocked, setup_data):
+        """Test para verificar que un sysadmin puede acceder a la vista de Superset"""
+
+        auth_headers = {
+            "Authorization": setup_data.sysadmin["token"],
+            "X-CSRF-Token": "test_csrf_token"
+        }
+        url = url_for('superset_blueprint.index')
+
+        print(f"🔹 URL de solicitud: {url}")
+        print(f"🔹 Headers enviados: {auth_headers}")
+
+        # Obtener los datos del usuario a través de user_show
+        try:
+            user_data = toolkit.get_action('user_show')({}, {'id': setup_data.sysadmin['name']})
+            print(f"🔍 Usuario detectado antes de la solicitud: {user_data}")
+        except Exception as e:
+            print(f"❌ Error al obtener datos del usuario en la prueba: {e}")
+            user_data = None
+
+        assert user_data, "No se encontró el usuario en CKAN"
+        assert user_data.get('sysadmin', False), "El usuario no tiene permisos de sysadmin"
+
+        response = app_httpx_mocked.get(url, headers=auth_headers)
+
+        assert response.status_code == 200, f"Esperado 200, pero recibió {response.status_code}"
+        assert 'Superset charts' in response.body, "No se encontró el texto esperado en la respuesta"
+
+    def test_index_non_sysadmin_cannot_access(self, app_httpx_mocked, setup_data):
+        """Test para verificar que un usuario no sysadmin no puede acceder a la vista de Superset"""
+        auth = {"Authorization": setup_data.user_regular['token']}
+        url = url_for('superset_blueprint.index')
+        response = app_httpx_mocked.get(url, extra_environ=auth, expect_errors=True)
```

Comment:
```suggestion
        response = app_httpx_mocked.get(url, headers=auth)
```

New CKAN version uses `headers` instead `extra_environ`
Not sure about `expect_errors=True`. Is really required? It look from old CKAN versions